### ISSUE TITLE
Handle previewing small images better

### ISSUE THREAD

**Issue by ericallam** [apihero-run/jsonhero-web#37]
For example, the image below is 64x64

<img width="675" alt="Screen Shot 2022-04-29 at 16 28 33" src="https://user-images.githubusercontent.com/534/165975921-a8dbc2b7-a99d-4891-b431-3b101563037c.png">

You can see it live here: https://jsonhero.io/j/REqfdqUzgoYP

**Comment by bolshoytoster** [comment]
@ericallam how do you mean? Isn't that how a 64x64 jpeg would be expected to look?

**Comment by ericallam** [comment]
Yes but I think it should probably be shown to be smaller, matching it's resolution
